<ui version="4.0" >
 <class>InformerPage</class>
 <widget class="QDialog" name="InformerPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblCreatePerson" >
     <property name="text" >
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblSubject" >
     <property name="text" >
      <string>Тема</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="lblCreatePersonValue" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="lblCreateDatetimeValue" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="lblSubjectValue" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="lblText" >
     <property name="text" >
      <string>Текст</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QTextBrowser" name="edtText" >
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QCheckBox" name="chkMarkViewed" >
     <property name="text" >
      <string>Больше не &amp;показывать</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblCreateDatetime" >
     <property name="text" >
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
